src/shell.ts

```diff
@@ -594,23 +596,19 @@ export class Shell {
         const fields = [
             { prompt: "Verify peer" },
             { prompt: "Not now" },
-            { prompt: "Don't ask again"}
+            { prompt: "Always verify"}
```

Comment:
It's better but still a bit weird. Let's remove this option for now. 

Please open an issue with a new feature targeted at those choosing "not now" and giving them a way to verify the peer.

Reply:
How would they verify?